solutionbox/code_free_ml/analyze_data.py

```diff
@@ -394,34 +497,37 @@ def preprocessing_fn(inputs):
               make_bag_of_words_tito(vocab=vocab, part='weights'),
               inputs[name])
         else:
+          # ONE_HOT_TRANSFORM: making a dense vector is done at training
+          # EMBEDDING_TRANSFROM: embedding vectors have to be done at training
           result[name] = tft.map(make_str_to_int_tito(vocab, len(vocab)),
                                  inputs[name])
-      elif transform_name == KEY_TRANSFORM:
-          result[name] = inputs[name]
       else:
         raise ValueError('unknown transform %s' % transform_name)
     return result
 
   return preprocessing_fn
 
 
-def make_tft_input_schema(schema, args):
-  """Make a tft-stype schema file.
+def make_tft_input_schema(schema, output_dir):
+  """Make a TFT Schema object
 
-  In the tft tramework, this is where default values are recoreded for training.
+  In the tft framework, this is where default values are recoreded for training.
 
   Args:
-    schema: schema file
-    args: command line args
+    schema: schema list
+    output_dir: output folder
+
+  Returns:
+    TFT Schema object.
   """
   result = {}
 
   # stats file us used to get default values.
   stats = {}
-  if file_io.file_exists(os.path.join(args.output_dir, STATS_FILE)):
+  if file_io.file_exists(os.path.join(output_dir, STATS_FILE)):
```

Comment:
The stats file should always exist according to your impl above. So remove the if?